plugins/MantisCoreFormatting/MantisCoreFormatting.php

```diff
@@ -131,9 +131,10 @@ function ( ) {
 
 		$t_string = MantisMarkdown::convert_text( $t_string );
 
-		# restore '<' converting &lt; within backticks 
-		$t_string = str_replace( '&amp;lt;', '&lt;', preg_replace( '#&amp;lt;.+?.*?&gt;#i', '$0', $t_string ));
-			
+		# restore '<' signs converting &lt; within code block tag (backticks)
+		# markdown coudnt perform well once the text procces executed the text
```

Comment:
2nd line of comment does not make sense. 
- coundt -> doesn'nt ?
- define what _perform well_ means
- _the text procces_ (processing ?) _executed the text_ what does that mean ?